### PR TITLE
add queues tab to details page

### LINKED ISSUE #18: [Queues page]: Build component to display queues data
Acceptance criteria:

- This component should be part of queues tab page
- Build react component
- Build react container component  and pass dummy data in component. Later we will integrate api for live data in separate task
- Write storybook component to see this components standalone
- Make sure component should be responsive and fit for all kind of view port

Wireframe: https://www.ssorj.net/brokerage/brokers/build-infra/index.html

### PR DESCRIPTION
**Pull request by AjayJagan**
Resolves #18

### CODE REVIEW

File: src/brokers/broker-details/components/Queues/Queues.component.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  isLoaded: boolean;
+  loadError: boolean;
+};
+
+const Queues: React.FC<QueuesProps> = ({ queueData, isLoaded, loadError }) => {
+  const { t } = useTranslation();
+
+  const columns: TableColumn<Queue>[] = [
+    {
+      title: t('Name'),
```
Comment (ajaypratap003) [inline src/brokers/broker-details/components/Queues/Queues.component.tsx:20]:
```suggestion
      title: t('name'),
```

File: src/brokers/broker-details/components/Queues/Queues.component.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      id: 'name',
+    },
+    {
+      title: t('Routing type'),
```
Comment (ajaypratap003) [inline src/brokers/broker-details/components/Queues/Queues.component.tsx:24]:
```suggestion
      title: t('routing_type'),
```

Please keep translations string key in lower case with snake format.

File: src/brokers/broker-details/components/Queues/Queues.component.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      id: 'routing_type',
+    },
+    {
+      title: t('Auto-create queues'),
```
Comment (ajaypratap003) [inline src/brokers/broker-details/components/Queues/Queues.component.tsx:28]:
```suggestion
      title: t('auto_create_queues'),
```

File: src/brokers/broker-details/components/Queues/Queues.component.tsx (hunk 2 above)
Comment (ajaypratap003) [reply src/brokers/broker-details/components/Queues/Queues.component.tsx:24]: Don't forget to add all the translation string in resource file. 
https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/blob/main/locales/en/plugin__activemq-artemis-self-provisioning-plugin.json

**Review by ajaypratap003 — CHANGES_REQUESTED** [review]
Adding few suggestions related to translation strings. Rest looks good. 
Good work.

File: src/brokers/broker-details/components/Queues/Queues.component.tsx (hunk 3 above)
Comment (AjayJagan) [reply src/brokers/broker-details/components/Queues/Queues.component.tsx]: done

File: src/brokers/broker-details/components/Queues/Queues.component.tsx (hunk 1 above)
Comment (AjayJagan) [reply src/brokers/broker-details/components/Queues/Queues.component.tsx]: done

**Review by ajaypratap003 — APPROVED** [review]

**Pull request merged**